packages/web-client/tests/integration/components/card-pay/merchant-payment-request-card-test.ts

```diff
@@ -0,0 +1,119 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import config from '@cardstack/web-client/config/environment';
+
+// selectors
+const BETA_ACCESS_LINK = '[data-test-payment-request-beta-access-link]';
+const MERCHANT = '[data-test-merchant]';
+const AMOUNT = '[data-test-payment-request-amount]';
+const USD_AMOUNT = '[data-test-payment-request-usd-amount]';
+const QR_CODE = '[data-test-styled-qr-code]';
+const DEEP_LINK = '[data-test-payment-request-deep-link]';
+const LINK_VIEW_TOGGLE = '[data-test-payment-request-link-view-toggle]';
+const PAYMENT_URL = '[data-test-payment-request-url]';
+
+// fixed data
+const amount = '300';
+const usdAmount = '3';
+const paymentURL =
+  'https://pay.cardstack.com/merchat-asdnsadkasd?id=0x1238urfds&amount=73298587423545';
+const merchant = {
+  name: 'Happii Creations',
+  logoBackground: 'cornflowerblue',
+  logoTextColor: 'black',
+};
+
+module(
+  'Integration | Component | card-pay/merchant-payment-request-card',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    hooks.beforeEach(function () {
+      this.setProperties({
+        amount,
+        usdAmount,
+        paymentURL,
+        merchant,
+      });
+    });
+
+    test('It renders the non-deep-link view correctly', async function (assert) {
+      await render(hbs`
+        <CardPay::MerchantPaymentRequestCard
+          @amount={{this.amount}}
+          @usdAmount={{this.usdAmount}}
+          @merchant={{this.merchant}}
+          @paymentURL={{this.paymentURL}}
+          @canDeepLink={{false}}
```

Comment:
Using an argument rather than a media query to determine whether to show the "Pay Merchant" link, because this depends more on platform than on things that are easily detected via media query.